src/rank_llm/retrieve_and_rerank.py

```diff
@@ -37,6 +37,10 @@ def retrieve_and_rerank(
 ):
     # Construct Rerank Agent
     if "gpt" in model_path or use_azure_openai:
+        from dotenv import load_dotenv

```

Comment:
This should not exist in the latest main, please make sure your base is updated.